backend/app/controllers/spree/admin/promotions_controller.rb

```diff
@@ -62,6 +63,10 @@ def promotion_code_batch_params
       def promotion_includes
         [:promotion_actions]
       end
+
+      def clear_boolean(query_params, condition)
+        query_params.delete(condition) if query_params[condition] == "0"
```

Comment:
Do we actually need this? The [Ransack documentation](https://github.com/activerecord-hackery/ransack#using-scopesclass-methods) states that scopes are only applied for `true` values (unless the scope accepts arguments), so a value of `0` should cause Ransack to ignore the scope.

Reply:
You're right! Will make the change